frontends/web/src/routes/account/account.tsx

```diff
@@ -332,18 +332,15 @@ class Account extends Component<Props, State> {
                             <div class="flex flex-row flex-between flex-items-center">
                                 <label className="labelXLarge">{t('accountSummary.availableBalance')}</label>
                                 <div className={style.actionsContainer}>
-                                    { this.supportsBuy() && (
-                                          <a href={`/account/${code}/buy`} className={style.buy}><span>{t('button.buy')}</span></a>
-                                    )
-                                    }
-                                    {
-                                        canSend ? (
-                                            <a href={`/account/${code}/send`} className={style.send}><span>{t('button.send')}</span></a>
-                                        ) : (
-                                            <a className={[style.send, style.disabled].join(' ')}><span>{t('button.send')}</span></a>
-                                        )
-                                    }
+                                    {canSend ? (
+                                        <a href={`/account/${code}/send`} className={style.send}><span>{t('button.send')}</span></a>
+                                    ) : (
+                                        <a className={[style.send, style.disabled].join(' ')}><span>{t('button.send')}</span></a>
```

Comment:
seems like these could be folded into one `<a href` and only change the `className` based on `canSend`. would read nicer imho.

Reply:
Yes argee, but this shouldn't be an anchor at all, changing to span + classes ok?